sr_gui_bootloader/src/sr_gui_bootloader/bootloader.py

```diff
@@ -88,11 +89,18 @@ def __init__(self, context):
         loadUi(ui_file, self._widget)
         self._widget.setObjectName('SrMotorResetterUi')
         context.add_widget(self._widget)
+        
+        #setting the prefixes
+        self._prefix = ""
+        self._widget.select_prefix.addItem("")
+        self._widget.select_prefix.addItem("rh/")
+        self._widget.select_prefix.addItem("lh/")
+        self._widget.select_prefix.currentIndexChanged['QString'].connect(self.prefix_selected)
 
         # motors_frame is defined in the ui file with a grid layout
         self.motors = []
         self.motors_frame = self._widget.motors_frame
-        self.populate_motors()
+        #self.populate_motors()
```

Comment:
@beatrizleon  Is there any reason why you have left commented code in the final PR? Is this for the future references? In general it would be good to have code without commented blocks.